docs/application/native/guides/text-input/voice-control-engine.md

```diff
@@ -0,0 +1,527 @@
+# Voice-Control-Engine
+
+
+Voice-Control-Engine (VCE) is an engine to recognize sound data. The user records this data and sends the result as a predefined command.
+
+VCE developers can provide VCE service users, to apply VCE with functions that is necessary to operate the engine.
+
+The main features of the VCE API include:
+
+- Preparing the VCE service for use
+
+  You can connect the VCE service application. [Basic VCE Processes](#basic) and [Prerequisites](#prerequisites), are defined.
```

Comment:
Please change:
You can connect the VCE service application. [Basic VCE Processes](#basic) and [Prerequisites](#prerequisites), are defined.
to 
You can connect the VCE service application. [Basic VCE Processes](#basic) and [Prerequisites](#prerequisites) are defined.
"Please can you remove comma before and"
Question: Is there any specific reason that the comma is retained in this sentence?

Reply:
Done.
Answer : No, it's a misstake. your suggestion looks good. thank you.